File: Envws/EnvwsTracker/TrackerJobData.cs
using EnvwsLib.DataContracts;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EnvwsTracker 
{
	/// <summary>
	/// Associates a completed JobData with a boolean that indicates if the JobData
    /// was successfully returned by the Tracker that executed it.
	/// </summary>
    public class TrackerJobData
    {
        public bool HasBeenReturnedToTracker { get; set; }
        public JobData TheJobInQuestion { get; set; }

		public TrackerJobData()
        {
            HasBeenReturnedToTracker = false;
        }
    }
}

File: java/Jenvws/service/src/main/java/Tracker/TrackerJobData.java
package Tracker;

import DataObjects.JobData;

/**
 *
 */
public class TrackerJobData {
    private boolean hasBeenReturnedToTracker = false;
    private JobData theJobInQuestion = JobData.emptyJob();

    public TrackerJobData() { }

    public boolean isHasBeenReturnedToTracker() {
        return hasBeenReturnedToTracker;
    }

    public void setHasBeenReturnedToTracker(boolean hasBeenReturnedToTracker) {
        this.hasBeenReturnedToTracker = hasBeenReturnedToTracker;
    }

    public JobData getTheJobInQuestion() {
        return theJobInQuestion;
    }

    public void setTheJobInQuestion(JobData theJobInQuestion) {
        this.theJobInQuestion = theJobInQuestion;
    }
}
